core/store/orm/orm_test.go

```diff
@@ -1357,3 +1357,29 @@ func TestJobs_ScopedInitiator(t *testing.T) {
 	expectation := []string{fmJob.ID.String(), twoInitrJob.ID.String()}
 	assert.ElementsMatch(t, expectation, actual)
 }
+
+func TestJobs_MoreThanBatchWithArchives(t *testing.T) {
```

Comment:
The naming could be better.  Maybe test SQL lite load?  We could also add a comment for rationale behind the test.

Reply:
The test isn't a test of SQLite load. It's a test that if you have more records than the batch size, and some are archived, it continues searching and doesn't exit early.